x-pack/plugins/apm/server/lib/rum_client/get_web_core_vitals.ts

```diff
@@ -124,13 +124,12 @@ export async function getWebCoreVitals({
     { value: 0, key: 0 },
   ];
 
-  // Divide by 1000 to convert ms into seconds
   return {
     cls: String(cls?.values['50.0']?.toFixed(2) || 0),
-    fid: ((fid?.values['50.0'] || 0) / 1000).toFixed(2),
-    lcp: ((lcp?.values['50.0'] || 0) / 1000).toFixed(2),
-    tbt: tbt?.values['50.0'] || 0,
-    fcp: fcp?.values['50.0'] || 0,
+    fid: fid?.values['50.0'] ?? 0,
```

Comment:
Could this value have > 2 decimal digits? Ignore if the answer is no, or formatting it as such is no longer important.

Reply:
it's done in frontend